src/Tribe/Admin/Notice/Marketing.php

```diff
@@ -36,179 +39,137 @@ public function __construct() {
+	 * Unix times for Monday of Thanksgiving week @ 6am UTC and Dec 1 2020 @ 6am UTC.
 	 * 6am UTC is midnight for TheEventsCalendar.com, which uses the America/Los_Angeles time zone.
 	 *
-	 * @since 4.7.23
+	 * @since TBD
 	 *
 	 * @return boolean
 	 */
-	public function bf_2018_should_display() {
-		$bf_sale_start = $this->get_bf_2018_start_time();
-		$bf_sale_end   = $this->get_bf_2018_end_time();
-
-		return $bf_sale_start <= time() && time() < $bf_sale_end;
-	}
-
-	/**
-	 * HTML for the Black Friday 2018 notice.
-	 *
-	 * @since 4.7.23
-	 *
-	 * @return string
-	 */
-	public function bf_2018_display_notice() {
-
-		Tribe__Assets::instance()->enqueue( array( 'tribe-common-admin' ) );
-
-		$mascot_url = Tribe__Main::instance()->plugin_url . 'src/resources/images/mascot.png';
-		$end_time   = $this->get_bf_2018_end_time();
-
-		ob_start();
-
-		if ( $this->tec_is_active && ! $this->et_is_active ) {
-			include Tribe__Main::instance()->plugin_path . 'src/admin-views/notices/tribe-bf-2018-tec.php';
-		} elseif ( $this->et_is_active && ! $this->tec_is_active ) {
-			include Tribe__Main::instance()->plugin_path . 'src/admin-views/notices/tribe-bf-2018-et.php';
-		} else {
-			include Tribe__Main::instance()->plugin_path . 'src/admin-views/notices/tribe-bf-2018-general.php';
+	public function black_friday_should_display() {
+		// If upsells have been manually hidden, respect that.
+		if ( defined( 'TRIBE_HIDE_UPSELL' ) && TRIBE_HIDE_UPSELL ) {
+			return false;
 		}
 
-		return ob_get_clean();
-	}
-
-	/**
-	 * Register the Gutenberg Release notice (November 2018).
-	 *
-	 * @since 4.7.23
-	 */
-	public function gutenberg_release_notice() {
+		$now           = Dates::build_date_object( 'now', 'UTC' )->format( 'U' );
+		$bf_sale_start = $this->get_black_friday_start_time();
+		$bf_sale_end   = $this->get_black_friday_end_time();
 
-		tribe_notice(
-			'gutenberg-release-2018',
-			array( $this, 'gutenberg_release_display_notice' ),
-			array(
-				'type'    => 'warning',
-				'dismiss' => 1,
-				'wrap'    => false,
-			),
-			array( $this, 'gutenberg_release_should_display' )
-		);
-	}
+		$current_screen = get_current_screen();
 
-	/**
-	 * Gets the end time for the Gutenberg release notice.
-	 *
-	 * @since 4.7.23
-	 *
-	 * @return int
-	 */
-	public function get_gutenberg_release_end_time() {
+		$screens = [
+			'tribe_events_page_tribe-app-shop', // App shop.
+			'tribe_events_page_tribe-common', // Settings & Welcome.
+		];
 
-		/**
-		 * Allows filtering of the default Gutenberg Release Notice's end time, mainly for testing purposes.
-		 *
-		 * @since 4.7.23
-		 *
-		 * @param int $gutenberg_release_end_time Defaults to Nov 17 2018 @ midnight, California time.
-		 */
-		return apply_filters( 'tribe_gutenberg_release_notice_end_time', 1542434400 );
-	}
+		// If not a valid screen, don't display.
+		if ( empty( $current_screen->id ) || ! in_array( $current_screen->id, $screens ) ) {
```

Comment:
```suggestion
		if ( empty( $current_screen->id ) || ! in_array( $current_screen->id, $screens, true ) ) {
```

Reply:
I think that tweak is a bit superfluous as the values being checked cannot be interpreted as anything other than strings. That third parameter really matters when dealing with truthy values, numbers, etc.

Reply:
I agree but phpcs catches it and every time I see it I enforce the third parameters so it never triggers a notice about it.